utility.js

```diff
@@ -5,3 +5,7 @@ export const sanitizeInput = (value) => {
 export const clearInputField = (inputElement) => {
     inputElement.value = "";
 };
+
+export const displayPropertySetter = (element, property) => {
+    element.style.display = property;
+};
```

Comment:
Setting style element is considered an anti pattern as it creates a lot of confusion and complicity later down the road.

Let's rethink this solution of adding css through class.
```js
export const addClass = (element, class) => {
   if(element.classList.contains(class){
       return
   }
   
   element.classList.add(class)
};
```

For remove class and toggle class, we can do it in line
```js
$element.classList.toggle("hide")
```
Then we can create atomic classes that will be added and removed from elements on condition.

```css
.show {
    display: block;
}

.hide {
   display: none;
}
```

Reply:
bhai I thought about using toggle function, but in our app there is some block element. So if I don't make them inline-block the ui structure will be changed at this point. 
So in this case, we can addCase or a custom toggler function. Which one will be better?

Reply:
Don't worry about UI structure. That will be handled when we style the whole app.

Reply:
okay bhai.